### PR TITLE
Adding support for Arc Appliance

### PR DESCRIPTION
**Pull request by willdavsmith**
* Added sample detector for Arc Appliance
* Added sample gist for Arc Appliance

### CODE REVIEW

File: data/templates/Detector_ArcAppliance.csx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+using System;
+using System.Threading;
+
+private static string GetQuery(OperationContext<ArmResource> cxt)
+{
+    return
+    $@"
+        let startTime = datetime({cxt.StartTime});
+        let endTime = datetime({cxt.EndTime});
+        cluster('ClusterName').database('DBName').YOUR_TABLE_NAME
```
Comment (nmallick1) [inline data/templates/Detector_ArcAppliance.csx:10]: Please change this line 
FROM cluster('ClusterName').database('DBName').YOUR_TABLE_NAME
To YOUR_TABLE_NAME

This is due to a recent change in the platform. The documentation has now been updated.

File: data/templates/Detector_ArcAppliance.csx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+}
+
+
+[ArmResourceFilter(provider: "Microsoft.ResourceConnector", resourceTypeName: "appliances")]
+[Definition(Id = "YOUR_DETECTOR_ID", Name = "", Author = "YOUR_ALIAS", Description = "")]
+public async static Task<Response> Run(DataProviders dp, OperationContext<ArmResource> cxt, Response res)
+{
+    res.Dataset.Add(new DiagnosticData()
+    {
+        Table = await dp.Kusto.ExecuteClusterQuery(GetQuery(cxt), null, "GetQuery"),
```
Comment (nmallick1) [inline data/templates/Detector_ArcAppliance.csx:23]: Please change this line 
FROM ==> await dp.Kusto.ExecuteClusterQuery(GetQuery(cxt), null, "GetQuery")
To ==> await dp.Kusto.ExecuteClusterQuery(GetQuery(cxt), "KustoClusterName", "KustoDatabaseName", null, "GetQuery")

This is due to a recent change in the platform. The documentation has now been updated.

**Review by nmallick1 — CHANGES_REQUESTED** [review]

File: data/templates/Detector_ArcAppliance.csx (hunk 1 above)
Comment (willdavsmith) [reply data/templates/Detector_ArcAppliance.csx]: Fixed in latest commit

File: data/templates/Detector_ArcAppliance.csx (hunk 2 above)
Comment (willdavsmith) [reply data/templates/Detector_ArcAppliance.csx]: Fixed in latest commit

**Review by nmallick1 — APPROVED** [review]

**Pull request merged**